solarforecastarbiter/datamodel.py

```diff
@@ -1418,19 +1418,22 @@ class PlotlyReportFigure(ReportFigure):
         A descriptive name for the figure.
     spec: str
         JSON string representation of the plotly plot.
-    svg: str
-        A static svg copy of the plot, for including in the pdf version.
     figure_type: str
         The type of plot, e.g. bar or scatter.
+    pdf: str
+        A static PDF copy of the plot, for including in PDF reports.
+    svg: str
+        DEPRECATED for pdf. A static svg copy of the plot.
```

Comment:
I don't quite understand. Is the attribute going to disappear in a future version?

Reply:
I hope so. Basically, new reports will have pdf only, but we still need to be able to load older ones into the datamodel.